cmd/snap/error.go

```diff
@@ -222,6 +222,31 @@ If you understand and want to proceed repeat the command including --classic.
 		isError = false
 		usesSnapName = false
 		msg = i18n.G("snapd is about to reboot the system")
+	case client.ErrorKindInsufficientSpace:
+		// this error carries multiple snap names
+		usesSnapName = false
+		values, ok := err.Value.(map[string]interface{})
+		if ok {
+			changeKind, _ := values["change-kind"].(string)
+			snaps, _ := values["snap-names"].([]interface{})
+			snapNames := make([]string, len(snaps))
+			for i := range snaps {
+				snapNames[i] = fmt.Sprintf("%q", snaps[i])
+			}
+			names := strings.Join(snapNames, ",")
```

Comment:
there's strutil.Quoted that does what the loop+Join does I think

Reply:
Indeed, thank you!